frontend/src/components/__tests__/ChartWidget.test.tsx

```diff
@@ -6,6 +6,23 @@ import { MemoryRouter } from "react-router-dom";
 
 jest.mock("../../hooks");
 
+const { ResizeObserver } = window;
```

Comment:
can you move  this to the jest.setup.ts file?

looks like this is always the same for all files